components/breadcrumbsv2/Breadcrumbs.module.scss

```diff
@@ -0,0 +1,138 @@
+@import '../../styles/index';
+
+.root {
+	@include flex(row, flex-start, center);
+	width: 100%;
+	height: 100%;
+
+	button[data-elem='base-cell'].selected[type='submit'] {
+		@include flex(row, flex-start, center);
+		height: 1.9375rem;
```

Comment:
Instead of using fixed height. Have that as min hight and use padding as per design